fdbserver/include/fdbserver/DDTxnProcessor.h

```diff
@@ -65,6 +70,10 @@ class IDDTxnProcessor : public ReferenceCounted<IDDTxnProcessor> {
 		return Void();
 	}
 
+	virtual Future<int> tryUpdateReplicasKeyForDc(const Optional<Key>& dcId, const int& storageTeamSize) const {
```

Comment:
Nit: IMO this should be marked as a pure virtual method, with a definition provided in `DDMockTxnProcessor`.

Reply:
The same applies for several other methods of `IDDTxnProcessor`.